src/hooks/useFeatureToggle.ts

```diff
@@ -10,7 +10,10 @@ import { Tiltakskode } from '../api/data/tiltak'
 
 let cachedFeatureToggles: FeatureToggles | undefined = undefined
 
-const tiltakstyperMedForslag = [Tiltakskode.ARBFORB]
+const tiltakstyperKometAlltidErMasterFor
+  = [Tiltakskode.ARBFORB]
+
+const tiltakstyperKometKanskjeErMasterFor: Tiltakskode[] = []
```

Comment:
Denne er der for at vi skal kunne legge inn de neste tiltakstypene vi skal ta over :)

Reply:
Hahaha navnet 😂  `kanskjeErMasterFor` 😂